src/contexts/APIRouterContext/Provider.tsx

```diff
@@ -244,13 +244,13 @@ const APIRouterProvider: React.FC = ({ children }) => {
   );
 
   const handleSubmitSecret = useCallback(async () => {
-    let result;
     try {
       if (afterSecretCB.current === undefined) {
         throw new Error(`handleSubmitSecret afterSecretCB.current is undefined: ${afterSecretCB.current}`);
       }
 
-      result = await afterSecretCB.current(userSecretInput);
+      // TODO: should consider doing something with the return of this function call
```

Comment:
i think the try/catch nature here is what we care about. if there is a problem, throw an error.

a return could also be helpful, unsure yet what it needs to be used for.

Reply:
yea same here, just seems like it needs something I just don't know what yet

Reply:
check if typescript will protect us by changing the functions for the callbacks to return a `Promise<undefined>` only. should help